troubleshooting/state-migration/exercise/build.gradle

```diff
@@ -17,13 +17,14 @@
 
 buildscript {
     repositories {
-        jcenter()
+        gradlePluginPortal()
+        mavenCentral()
```

Comment:
For further references, can you add a sentence why this replacement is needed?

Reply:
updated commit message. Thanks!